app/uploaders/file_uploader.rb

```diff
@@ -49,7 +49,10 @@ def cache_dir
         tmp = Tempfile.new 'op_uploaded_files'
         path = Pathname(tmp)
 
-        tmp.delete # delete temp file
+        tmp.unlink # delete temp file. In windows this fails silently.
```

Comment:
@machisuji that would be fine. Actually "tmp.close!" alone will both delete (unlink) and close the temp file, so no need to delete it at all, prior or after. But I can't test the behavior in anyt other than Windows ATM.